tests/cpp/test_resize.cpp

```diff
@@ -6005,6 +6005,50 @@ TEST_F(ResizeTest, VectorizeSliceMultiplePaths) {
   EXPECT_EQ(tv6->getLoopDomain().back()->extent()->evaluate(), 2);
 }
 
+// The current analysis is not precise enough to pass this test
```

Comment:
In this test, tv0 is resized in two different ways. The spanning-tree based analysis is not guaranteed to correctly identify the vectorization constraint.

The WAR when applied to this case is still too conservative.